samples/charts/financial-chart/annotations/Pages/FinancialChartAnnotations.razor

```diff
@@ -1,95 +1,94 @@
 ﻿@page "/CategoryChartOverview"
 @* this namespace is required for Infragistics controls *@
 @using Infragistics.Blazor.Controls
+@using Services
 @inject IInfragisticsBlazor InfragisticsBlazor;
+@inject Samples.Services.StockItem StockItem;
+@inject Samples.Services.StockList StockList;
```

Comment:
remove injection of data models because they are not services!
replace `@using Services` with ```@using Samples.Services```